### ISSUE TITLE
minimind模型的网络运行架构图，可供参考！！！

### ISSUE THREAD

**Issue by PPalouga** [jingyaogong/minimind#222]
minimind模型的网络运行架构图绘制，可供参考！！！
![Image](https://github.com/user-attachments/assets/37d274e4-4234-4eb8-a8b8-9b0dc96054f2)

**Comment by jingyaogong** [comment]
🤣好长，谢谢！

**Comment by xiaomukuaier** [comment]
请教这种架构是纯手动梳理出来的吗，太厉害了！以及请问用的是什么软件画的，还很好看。

**Comment by jingyaogong** [comment]
> 请教这种架构是纯手动梳理出来的吗，太厉害了！以及请问用的是什么软件画的，还很好看。

可以了解一下onnx模型的可视化

**Comment by PPalouga** [comment]
我用的torchviz 
from torchviz import make_dot
MyConvNetVis = make_dot(y.logits, params=dict(list(model.named_parameters()) + [('x', x)]))
预测运行一遍自动添值推理
